instrumentation/lettuce-4.3/src/test/java/com/nr/lettuce43/instrumentation/helper/Data.java

```diff
@@ -0,0 +1,11 @@
+package com.nr.lettuce43.instrumentation.helper;
```

Comment:
Test source files require the copyright header as well.

Reply:
oh, ok...didn't know test files too...will do!